src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/7.diff.sql

```diff
@@ -0,0 +1,397 @@
+CREATE TYPE dbo.ReferenceSearchParamTableType_2 AS TABLE
+(
+    SearchParamId smallint NOT NULL,
+    BaseUri varchar(128) COLLATE Latin1_General_100_CS_AS NULL,
+    ReferenceResourceTypeId smallint NULL,
+    ReferenceResourceId varchar(64) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    ReferenceResourceVersion int NULL
+);
+
+ALTER TABLE dbo.ReferenceSearchParam
+    ALTER COLUMN ReferenceResourceTypeId smallint NULL;
+
+CREATE TYPE dbo.ReferenceTokenCompositeSearchParamTableType_2 AS TABLE
+(
+    SearchParamId smallint NOT NULL,
+    BaseUri1 varchar(128) COLLATE Latin1_General_100_CS_AS NULL,
+    ReferenceResourceTypeId1 smallint NULL,
+    ReferenceResourceId1 varchar(64) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    ReferenceResourceVersion1 int NULL,
+    SystemId2 int NULL,
+    Code2 varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL
+)
+
+ALTER TABLE dbo.ReferenceTokenCompositeSearchParam
+    ALTER COLUMN ReferenceResourceTypeId1 smallint NULL;
+
+GO
+
+/*************************************************************
+    Stored procedures for creating and deleting
+**************************************************************/
+
+--
+-- STORED PROCEDURE
+--     UpsertResource
```

Comment:
>UpsertResource [](start = 7, length = 14)

should we change name to `UpsertResource_2` ?

Reply:
Good catch. Renamed the name in the the comments.